code/user_settings.py

```diff
@@ -56,95 +50,11 @@ def _load_csv_dict(
                 output, spoken_form = row[:2]
                 if len(row) > 2:
                     print(
```

Comment:
Ditto re logging.warning.

Reply:
if this is useful for you I would vote to write up an issue,  and we can address this more broadly.   I generally have very little output,  so haven't bothered